pubapi/examples/PubAPIAuthenticator.java

```diff
@@ -0,0 +1,80 @@
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Example class for demonstrating ERCOT Public API authentication and endpoints/
+ * This uses the vanilla java.net.http packages for demonstration. Using other request methods 
+ * in Java will require different code but the endpoints are still identical
+ */
+public class PubAPIAuthenticator {
+
+    // USER MUST PROVIDE THIS INFORMATION 
+    private final static String USERNAME = "username";
+    private final static String PASSWORD = "password";
+    private final static String SUBSCRIPTION_KEY = "subscription-key";
+
+    /**
+     * Main method
+     * @param args unused
+     */
+    public static void main(String[] args) {
+
+        // HTTP Client
+        HttpClient httpClient = HttpClient.newHttpClient();
+
+        // Authorization URL ffor signing into ERCOT Public API account
+        final String AUTH_URI = "https://ercotb2c.b2clogin.com/ercotb2c.onmicrosoft.com/B2C_1_PUBAPI-ROPC-FLOW/oauth2/v2.0/token"
+        + "?username=" + USERNAME
+        + "&password=" + PASSWORD
+        + "&grant_type=password"
+        + "&scope=openid+fec253ea-0d06-4272-a5e6-b478baeecd70+offline_access"
```

Comment:
I thought we should leave them in their because these are supposed to be used by publicly by people who will be calling these endpoints themselves.

@osoberanis-ercot What do you think?

Reply:
Yes, we intent to do this.